yt/data_objects/tests/test_profiles.py

```diff
@@ -396,3 +397,42 @@ def test_index_field_units():
                       weight_field=None)
     assert str(prof['index', 'cell_volume'].units) == 'code_length**3'
     assert str(prof['gas', 'cell_volume'].units) == 'cm**3'
+
+
```

Comment:
I think you want the `@requires_module()` decorator for both astropy and pandas here.